float32 depth
float32 pitch
float32 speed
float32 depth_demand
float32 error
float32 CSint_error
float32 Tint_error
float32 der_error
float32 min_int_error
float32 max_int_error
float32 CL
float32 buoyancy
float32 CS_pitch_Pgain
float32 CS_pitch_Igain
float32 CS_pitch_Pmin
float32 CS_pitch_Pmax
float32 CS_pitch_Imin
float32 CS_pitch_Imax
float32 CS_pitch_min
float32 CS_pitch_max
float32 CS_Pgain
float32 CS_Igain
float32 CS_min 
float32 CS_max 
float32 CS_ss_pitch_angle
float32 CS_pitch_Pterm
float32 CS_pitch_Iterm
float32 CS_pitch_demand
float32 pitch_error
float32 CS_Pterm
float32 CSp
float32 CSs
float32 estLiftForce
float32 Thrust_Pgain
float32 Thrust_Igain
float32 Thrust_Dgain
float32 Thrust_Pmin
float32 Thrust_Pmax
float32 Thrust_Imin
float32 Thrust_Imax
float32 Thrust_Dmin
float32 Thrust_Dmax
float32 Thrust_Smin
float32 Thrust_Smax  
float32 Thrust_Pterm
float32 Thrust_Iterm
float32 Thrust_Dterm
float32 Thrust_total
int32 thruster0
int32 thruster1